recipe/meta.yaml

```
@@ -1,59 +1,61 @@
 {% set name = "spacy" %}
-{% set version = "3.5.3" %}
+{% set version = "3.7.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 35971d6721576538d6c423c66a09ce00bf66e10e40726a57b7a81993180c248c
+  sha256: cedf4927bf0d3fec773a6ce48d5d2c91bdb02fed3c7d5ec07bdb873f1126f1a0
+  patches:
+    - patches/0001-skip-beam-density-test.patch
 
 build:
   number: 0
   # no s390x support
-  skip: True  # [py<36 or (linux and s390x)]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
+  skip: True  # [py<36 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   entry_points:
     - spacy = spacy.cli:setup_cli
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - patch       # [not win]
+    - m2-patch    # [win]
   host:
+    - pip
     - python
-    - cymem 2.0.6
+    - numpy {{ numpy }}
+    - setuptools
+    - wheel
     - cython 0.29.32
```

```suggestion
    - cython >=0.25,<3.0
```
https://github.com/explosion/spaCy/blob/v3.7.2/pyproject.toml#L4C12-L4C23

---

ok, done,
why don't we exactly pin `cython`?

---

Well, you can use `0.29` but not `0.29.32` to avoid build-out issues for new Python versions.